### DISCUSSION TITLE
[Q/A][Java SDK (Component)] Consumer register-mode config cannot work

### DISCUSSION THREAD

**Discussion by Floweryu** [apache/dubbo#14725] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

I configured register-mode as ALL in consumers，like this：
<img width="998" alt="image" src="https://github.com/user-attachments/assets/631ff2c8-07f3-4065-9e47-37c619a8ee3d">

But I found that the instance of consumer registration seems to have nothing to do with the register-mode parameter，
![image](https://github.com/user-attachments/assets/c558bfab-8617-4316-97b6-961ef301814a)

Is this parameter invalid in consumers?  I think there are examples of the official document here. Can you help me answer it?
![image](https://github.com/user-attachments/assets/f6cb5b41-ba33-45d8-89cb-e5ff73234e70)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
Yes, only works in provider side
